packages/components/sronly/src/SrOnly.tsx

```diff
@@ -0,0 +1,27 @@
+import { cx } from 'emotion';
+import React from 'react';
+import { getStyles } from './SrOnly.styles';
+import type { CommonProps } from '@contentful/f36-core';
+
+export interface SrOnlyProps extends CommonProps {
+  children?: React.ReactNode;
+}
+
+export const SrOnly = ({
+  children,
+  className,
+  testId = 'cf-ui-sronly',
+  ...otherProps
+}: SrOnlyProps) => {
+  const styles = getStyles();
+
+  return (
+    <div
+      {...otherProps}
+      data-testid={testId}
```

Comment:
It might be easier to use `<Box>` here. We use `data-test-id` everywhere else.

Reply:
Yes about the `data-test-id `but not sure if it is a good idea to create a dependency to the `<Box>` component for a helper component like this?

Reply:
Another idea would be, instead of having `ScreenReaderOnly` as a dedicated component to extend `Box` component with "screen reader only" prop?

Reply:
oh I made a comment in package.json
A third option is to make this part of `f36-core` which is the package with `Box`

Reply:
I like option 3, what do the others think?

Reply:
@gui-santos @denkristoffer i moved it now to be part of the core package.